happi/backends/json_db.py

```diff
@@ -14,6 +13,14 @@
 logger = logging.getLogger(__name__)
 
 
+try:
+    import fcntl
+    on_unix = True
```

Comment:
👍 but you might consider saying `if fcntl is not None` and not having an extra state variable

Reply:
🤔 makes sense